### PR TITLE
Fix CPU/nestedvm testcases

### PR DESCRIPTION
**Pull request by dsrivastavv**
Look into commits for details

### CODE REVIEW

**Comment by dsrivastavv** [comment]
@LiliDeng FIO installation is getting stuck in dpkg update. Have you seen this before?
```
2022-04-25 23:39:58.147[10988][DEBUG] lisa.L2-VM[0].cmd[2003] cmd: ['sudo', 'dpkg', '--force-all', '--configure', '-a'], cwd: None, shell: False, sudo: True, posix: True, remote: True
2022-04-25 23:39:58.240[25040][DEBUG] lisa.L2-VM[0].cmd[2003].stdout dpkg: error: dpkg frontend is locked by another process
2022-04-25 23:39:58.243[10988][DEBUG] lisa.L2-VM[0].cmd[2003] execution time: 0.096 sec, exit code: 2
2022-04-25 23:39:58.247[10988][DEBUG] lisa.L2-VM[0].cmd[1261] cmd: ['pidof', 'dpkg', 'dpkg-deb'], cwd: None, shell: False, sudo: False, posix: True, remote: True
2022-04-25 23:39:58.322[10988][DEBUG] lisa.L2-VM[0].cmd[1261] execution time: 0.074 sec, exit code: 1
2022-04-25 23:39:58.323[10988][DEBUG] lisa.L2-VM[0].os found system dpkg process, waiting it...
2022-04-25 23:39:59.342[10988][DEBUG] lisa.L2-VM[0].cmd[9820] cmd: ['sudo', 'dpkg', '--force-all', '--configure', '-a'], cwd: None, shell: False, sudo: True, posix: True, remote: True
2022-04-25 23:39:59.388[18856][DEBUG] lisa.L2-VM[0].cmd[9820].stdout dpkg: error: dpkg frontend is locked by another process
2022-04-25 23:39:59.391[10988][DEBUG] lisa.L2-VM[0].cmd[9820] execution time: 0.044 sec, exit code: 2
2022-04-25 23:39:59.392[10988][DEBUG] lisa.L2-VM[0].cmd[4446] cmd: ['pidof', 'dpkg', 'dpkg-deb'], cwd: None, shell: False, sudo: False, posix: True, remote: True
2022-04-25 23:39:59.477[10988][DEBUG] lisa.L2-VM[0].cmd[4446] execution time: 0.084 sec, exit code: 1
2022-04-25 23:40:00.481[10988][DEBUG] lisa.L2-VM[0].cmd[1054] cmd: ['sudo', 'dpkg', '--force-all', '--configure', '-a'], cwd: None, shell: False, sudo: True, posix: True, remote: True
2022-04-25 23:40:00.579[24940][DEBUG] lisa.L2-VM[0].cmd[1054].stdout dpkg: error: dpkg frontend is locked by another process
2022-04-25 23:40:00.580[10988][DEBUG] lisa.L2-VM[0].cmd[1054] execution time: 0.094 sec, exit code: 2
2022-04-25 23:40:00.581[10988][DEBUG] lisa.L2-VM[0].cmd[278] cmd: ['pidof', 'dpkg', 'dpkg-deb'], cwd: None, shell: False, sudo: False, posix: True, remote: True
2022-04-25 23:40:00.661[10988][DEBUG] lisa.L2-VM[0].cmd[278] execution time: 0.079 sec, exit code: 1
2022-04-25 23:40:01.673[10988][DEBUG] lisa.L2-VM[0].cmd[7164] cmd: ['sudo', 'dpkg', '--force-all', '--configure', '-a'], cwd: None, shell: False, sudo: True, posix: True, remote: True
2022-04-25 23:40:01.718[10756][DEBUG] lisa.L2-VM[0].cmd[7164].stdout dpkg: error: dpkg frontend is locked by another process
2022-04-25 23:40:01.721[10988][DEBUG] lisa.L2-VM[0].cmd[7164] execution time: 0.047 sec, exit code: 2
2022-04-25 23:40:01.722[10988][DEBUG] lisa.L2-VM[0].cmd[3534] cmd: ['pidof', 'dpkg', 'dpkg-deb'], cwd: None, shell: False, sudo: False, posix: True, remote: True
2022-04-25 23:40:01.829[10988][DEBUG] lisa.L2-VM[0].cmd[3534] execution time: 0.106 sec, exit code: 1
2022-04-25 23:40:02.844[10988][DEBUG] lisa.L2-VM[0].cmd[1579] cmd: ['sudo', 'dpkg', '--force-all', '--configure', '-a'], cwd: None, shell: False, sudo: True, posix: True, remote: True
2022-04-25 23:40:02.889[12856][DEBUG] lisa.L2-VM[0].cmd[1579].stdout dpkg: error: dpkg frontend is locked by another process
2022-04-25 23:40:02.892[10988][DEBUG] lisa.L2-VM[0].cmd[1579] execution time: 0.048 sec, exit code: 2
2022-04-25 23:40:02.893[10988][DEBUG] lisa.L2-VM[0].cmd[1676] cmd: ['pidof', 'dpkg', 'dpkg-deb'], cwd: None, shell: False, sudo: False, posix: True, remote: True
2022-04-25 23:40:02.976[10988][DEBUG] lisa.L2-VM[0].cmd[1676] execution time: 0.082 sec, exit code: 1
2022-04-25 23:40:03.991[10988][DEBUG] lisa.L2-VM[0].cmd[114] cmd: ['sudo', 'dpkg', '--force-all', '--configure', '-a'], cwd: None, shell: False, sudo: True, posix: True, remote: True
2022-04-25 23:
```

File: lisa/tools/lsblk.py [hunk 1]
```diff
@@ -171,3 +171,19 @@ def get_disks(self, force_run: bool = False) -> List[DiskInfo]:
             )
 
         return disks
+
+    def find_disk_with_mountpoint(
```
Comment (squirrelsc) [inline lisa/tools/lsblk.py]: `find_disk_by_mountpoint`

File: microsoft/testsuites/performance/nestedperf.py [hunk 2]
```diff
@@ -585,61 +596,83 @@ def _storage_perf_qemu(
...
+                disk_count=l1_data_disk_count,
+                disk_setup_type=DiskSetupType.raid0,
+                disk_type=DiskType.premiumssd,
+                environment=environment,
+                num_jobs=num_jobs,
+                size_gb=8,
+                overwrite=True,
+            )
+        finally:
+            node.tools[Qemu].delete_vm()
```
Comment (squirrelsc) [inline microsoft/testsuites/performance/nestedperf.py:674]: If it's shared by all cases in this class, it can be written in `after_case`.

**Comment by squirrelsc** [comment]
> @LiliDeng FIO installation is getting stuck in dpkg update. Have you seen this before?
> 

It may happen, if the system is running update progress. That's why we have this logic. Does it end later? If it ended later, it's expected.

**Comment by dsrivastavv** [comment]
> > @LiliDeng FIO installation is getting stuck in dpkg update. Have you seen this before?
> 
> It may happen, if the system is running update progress. That's why we have this logic. Does it end later? If it ended later, it's expected.

This results in a random behaviour. Sometimes it does stop later, but in most cases the testcase times out

File: microsoft/testsuites/performance/nestedperf.py (hunk 2 above)
Comment (dsrivastavv) [reply microsoft/testsuites/performance/nestedperf.py:674]: It's not. We have both hyperv and qemu testcases here. There are functions to stop VM in Qemu and Hyperv./ But maybe I can do a platform check here and call the functions accordingly

**Comment by squirrelsc** [comment]
> > > @LiliDeng FIO installation is getting stuck in dpkg update. Have you seen this before?
> > 
> > 
> > It may happen, if the system is running update progress. That's why we have this logic. Does it end later? If it ended later, it's expected.
> 
> This results in a random behaviour. Sometimes it does stop later, but in most cases the testcase times out

Please share me a log, when it's timed out.

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**